### ISSUE TITLE
new setup: admin login fails with no message

### ISSUE THREAD

**Issue by Helmi** [Mailu/Mailu#1145]
Hey everyone,

first of all thanks for this very interesting project and your hard work on it.

I'm just doing my first setup of MailU so please bear with me if I'm doing some stupid mistakes. I think I have written the docs good enough and hope I didn't miss anything.

I fail to be able to login to the admin. I have tried the automatic user generation by setting env variables in the mailu.env file and I also tried the manual command to do so. I restarted everything several times (using compose flavor on 1.7).

Result is i'm not even getting an error message when trying to login on mydomain.com/admin - not sure if this is the way it should be (to not show an error) but I assume user generation has still failed somehow. I scrolled through the log outputs of the admin container but didn't find anything about a user being generated or not. I played with INITIAL_ADMIN_MODE and set it to create and I did get the exception expected. At least it was an exception from python (UNIQUE constraint failed user.email) that sounded right.

Still no error oin the frontend when logging in, also no error in the admin container's logs. I keep seeing the login form.

Version: 1.7, docker-compose 1.24.1 on Ubuntu 18.04 VPS w/ 4GB Ram

**Comment by kaiyou** [comment]
When you write no error, what do you mean exactly? Do you end up on the login page with again, do you get a white page?

Could you dump a couple lines of logs from both the admin and front container? Also, please check using your favourite db client that the user table is indeed populated.

Final question just in case: you have cookies enabled on your browser, right?

**Comment by Helmi** [comment]
I got it fixed eventually. For some reason my password (generated, 16 digits, only special char was <) didn't work - i changed it to 123456, removed the users from the sqlite db and then it worked. Not sure what the issue could have been with the PW but looks like it has been exactly that.

And yes by no error I mean I got back to the login form again with no message - as if just nothing happened, I only saw the loading indicator of the browser quickly showing up. 

The user table had been populated but I assume something with the PW didn't make it to the db correctly.
